man/bundle-remove.ronn

```diff
@@ -0,0 +1,25 @@
+bundle-remove(1) -- Removes gem specified in Gemfile
+===========================================================================
+
+## SYNOPSIS
+
+`bundle remove [GEMS] [--install]`
+
+## DESCRIPTION
+
+Removes a set of specified gems from the Gemfile or if needed from the bundle itself.
```

Comment:
> if needed from the bundle itself

This won't make a huge amount of sense to a lot of people. I would instead add a paragraph that explains the `--install` option.